integration_tests/package-lock.yml

```diff
@@ -0,0 +1,9 @@
+packages:
+- local: ../
+- package: fivetran/fivetran_utils
+  version: 0.4.8
+- package: dbt-labs/spark_utils
+  version: 0.3.0
+- package: dbt-labs/dbt_utils
+  version: 1.1.1
+sha1_hash: 570155077da10b186bf3640e96c2b6746d0fc6dc
```

Comment:
Let's add this to the `.gitignore`. I worry this will not pick up changes on subsequent runs. I would rather the integration tests just rebuilds this everytime.